docs/cql/ddl.rst

```diff
@@ -109,9 +109,12 @@ For example:
 
 .. TODO Add a link to the description of minimum_keyspace_rf when the ScyllaDB options section is added to the docs.
 
-You can configure the minimum acceptable replication factor using the ``minimum_keyspace_rf`` option. 
-Attempting to create a keyspace with a replication factor lower than the value set with 
-``minimum_keyspace_rf`` will return an error (the default value is 0). 
+* The replication factor configured for a DC should be equal to or higher than
```

Comment:
`higher` -> `lower`
We don't want to set a replication factor that is too high and can't be satisfied with the existing number of nodes.